src/Routes.tsx

```diff
@@ -18,7 +18,8 @@ import { ROUTE } from './route.enum';
 const GetStarted = React.lazy(() => import('./pages/GetStarted'));
 const WorkspacesList = React.lazy(() => import('./containers/WorkspacesList'));
 const WorkspaceDetails = React.lazy(() => import('./containers/WorkspaceDetails'));
-const IdeIframe = React.lazy(() => import('./pages/IdeIframe'));
+const IdeLoader = React.lazy(() => import('./containers/IdeLoader'));
+const LoadFactory = React.lazy(() => import('./containers/LoadFactory'));
```

Comment:
WDYT about renaming this container to `FactoryLoader` in order to align it with `IdeLoader`?

Reply:
Good point. I have fixed.